v5.17.2 (#6086)
mui/mui-x: CHANGELOG.md

All notable changes to this project will be documented in this file.
See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.

## 5.17.2

_Sep 8, 2022_

We'd like to offer a big thanks to the 3 contributors who made this release possible. Here are some highlights ✨:

### `@mui/x-data-grid@v5.17.2` / `@mui/x-data-grid-pro@v5.17.2` / `@mui/x-data-grid-premium@v5.17.2`

#### Changes

> - [DataGridPremium] Export column grouping in Excel (#5895) @alexfauquette

Author: @alexfauquette do you think this one is worth highlighting ?

Reviewer: Since we have nothing else, why not

> ### `@mui/x-date-pickers@v5.0.1` / `@mui/x-date-pickers-pro@v5.0.1`

Author: For v6 we should always release all our stable packages with the same version, but I think that for 1 patch before the alpha it's not worth jumping the pickers to 5.17.2

Author: No strong opinion here
I just think bumping from v5.0.0 to v5.17.2 to v6.0.0-alpha.1 can be confusing.
We probably should have bumped to v5.17.0 instead of 5.0.0 last week.

Reviewer 2: > We probably should have bumped to v5.17.0 instead of 5.0.0 last week.

Yes, that's my thinking as well, if we did not align the version with the "stable" release - no point in doing that now.

Reviewer 3: I was about to leave a new comment until I found this thread. I agree with the conclusion 👍. If we need to release more versions, it will be clearer with a `@mui/x-date-pickers-pro@v5.17.2`. So it can look like this in people's `package.json`:

```json
  "@mui/x-data-grid": "5.17.2",
  "@mui/x-date-pickers": "5.17.2",
```

All notable changes to this project will be documented in this file.
See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.

## 5.17.2

_Sep 8, 2022_

We'd like to offer a big thanks to the 3 contributors who made this release possible. Here are some highlights ✨:

- 🐞 Bugfixes

> ### `@mui/x-data-grid@v5.17.2` / `@mui/x-data-grid-pro@v5.17.2` / `@mui/x-data-grid-premium@v5.17.2`

Reviewer: I assume that's what @LukasTy  is proposing to be consistent with the unmodified community and pro

```suggestion
### `@mui/x-data-grid-premium@v5.17.2`
```

Author: Matheus pushed a fix on the Community package so the discussion is closed I guess
But I agree :+1:

All notable changes to this project will be documented in this file.
See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.

## 5.17.2

_Sep 9, 2022_

This release will the last regular release for our packages in version `v5.x`.

> We are starting the alpha process for the `v6.x` of our packages.

Reviewer 4: ```suggestion
From now on, we'll be focusing on developing MUI X v6. 
You can check the [roadmap](https://github.com/mui/mui-x/projects/1) for more details on what's coming next.

And if you'd like to help, please consider volunteering to give us a [user interview](https://docs.google.com/forms/d/11L_zxL7oD_B-ZrZDuSyIkUzJLzOTUU2M4vHXxMVtWhU/edit).
We'd love to know more about your use cases, pain points and expectations for the future.
```

> The `v5.x` will now only receive bugfixes.

Reviewer 5: ```suggestion
The `v5.x` will now only receive critical bug fixes.
```

> The `v5.x` will now only receive critical bug fixes.

Reviewer 4: ```suggestion
The `v5` packages will only get new versions to patch critical bug fixes.
```

All notable changes to this project will be documented in this file.
See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.

## 5.17.2

_Sep 9, 2022_

This release will the last regular release for our `v5` packages.
From now on, we'll be focusing on developing MUI X v6.
You can check the [roadmap](https://github.com/mui/mui-x/projects/1) for more details on what's coming next.

> And if you'd like to help, please consider volunteering to give us a [user interview](https://docs.google.com/forms/d/11L_zxL7oD_B-ZrZDuSyIkUzJLzOTUU2M4vHXxMVtWhU/edit).

Reviewer 4: ```suggestion
And if you'd like to help, please consider volunteering to give us a [user interview]([contacting us](https://forms.gle/vsBv6CLPz9h57xg8A)).
```
